pyleco/test.py

```diff
@@ -0,0 +1,118 @@
+#
```

Comment:
should this file maybe better live in the `tests` directory? Does this have to be in the main code tree/package proper?

Reply:
If you want to import it, it has to be in `pyleco`, similarly to `expected_protocol` in pymeasure.

Reply:
It contains helpful tools for testing (fake sockets etc.)